### ISSUE TITLE
Offer Option to Turn Off Swipe Feature

### ISSUE THREAD

**Issue by NonITExpert** [hariettem/Artemis#352]
Currently swiping to vote up or down is not working well in this update. I recommend allowing users to choose whether to turn swipe features on or off. If they are off, show up and down arrows, boost and save and reply buttons.

**Comment by RheingoldRiver** [comment]
Gonna mark this as low prio cos the higher prio thing is to fix the bug, this is a more long-term feature

**Comment by hariettem** [comment]
@NonITExpert when you say is not working, do you mean the vote actions error out? And the reply window doesn’t open?

**Comment by NonITExpert** [comment]
I have not been able to downvote or boost using swipe from right. It's my understanding that to downvote you need to long-pull from right to left. Every time I do, I am met with the reply screen.  Swipe from left to right to upvote works with no issues.

https://github.com/hariettem/Artemis/assets/138516886/18e0a0a7-05e1-46aa-afe3-96b8aa414a0e

**Comment by hariettem** [comment]
gotcha, does pressing the downvote button work after you open the menu?

**Comment by NonITExpert** [comment]
That's just it. I can find no other way to downvote

**Comment by hariettem** [comment]
If you want to swipe to downvote, you can click the dots menu (next to clock) or tap and hold on iOS (coming to android soon)

**Comment by NonITExpert** [comment]
Yeah, I'm on android  When I click the 3 dots next to clock, nothing happens...

**Comment by hariettem** [comment]
That's a bug, let me see what's up with that 👀

**Comment by NonITExpert** [comment]
I noticed screenshot happens after I turned off 'Link Boosts & Upvote', since I don't necessarily want each upvote boosted. Maybe Boost could have its own icon to click when you want to boost? Otherwise downvoting is not possible with boost/upvote turned on.

**Comment by NonITExpert** [comment]
Looks like a fix was made available in the latest release pushed out this evening that fixes this. Now in place of swipe I can click the 3 dots to the left of the 'clock' to upvote, downvote, boost etc. It works great. Thank you. Here's screenshot
![Screenshot_20230807_185809_Artemis](https://github.com/hariettem/Artemis/assets/138516886/4bdf823a-4296-4972-a4c7-a57a545a7477)

**Comment by RheingoldRiver** [comment]
great! I'll leave the ticket open as I do think this is a good feature to add (eventually)
